coalib/settings/Section.py

```diff
@@ -62,7 +62,7 @@ def __init__(self,
         self.contents = OrderedDict()
 
     def bear_dirs(self):
-        bear_dirs = path_list(self.get("bear_dirs", ""))
+        bear_dirs = glob_list(self.get("bear_dirs", ""))
```

Comment:
Moreover, the only change is the use of `glob_path` instead, which has tests of its own.